### PR TITLE
NOISSUE: added changelog generator script

### PR DESCRIPTION
**Pull request by ita-sammann**
Added changelog generator script.
Requires Python 3, Jinja 2, requests, logzero.
```
pip3 install jinja2 requests logzero
``` 

Usage:
```
scripts/changelog.py v0.9.0-rc3 v0.9.0-rc4 -e <your email> -t <JIRA api token> --html=.artifacts/release.html
```

You can generate the token here: https://id.atlassian.com/manage/api-tokens

### CODE REVIEW

File: scripts/changelog.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    file.write(html)
+    file.close()
+
+
+def render_html(context):
+    tpl = jinja2.Environment().from_string(page_tpl())
+    return tpl.render(context)
+
+
+def page_tpl():
```
Comment (dmitry-panchenko) [inline scripts/changelog.py]: Why do you use function here? Make it constant

File: scripts/changelog.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        key = i['fields']['status']['statusCategory']['key']
+        if key in status_colors:
+            i['fields']['status']['statusCategory']['status'] = status_colors[key]
+    return context
+
+
+def save_html(context, filename):
+    context = prepare_context(context)
+    html = render_html(context)
+    file = open(filename, 'w+')
```
Comment (dmitry-panchenko) [inline scripts/changelog.py]: `with` statement?

File: scripts/changelog.py [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+            issues.append(r.json())
+        except ValueError:
+            logger.error(f'Failed to get info for issue {key}')
+    return issues
+
+
+def print_changelog(current, new, issues):
+    print(f'Changelog from {current} to {new}:')
+    for i in issues:
+        issue = i
```
Comment (dmitry-panchenko) [inline scripts/changelog.py]: Why not just `for issue in issues `?

File: scripts/changelog.py [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+"""
+
+import argparse
+import re
+import subprocess
+import sys
+
+import jinja2
+import requests
+from logzero import logger
```
Comment (dmitry-panchenko) [inline scripts/changelog.py]:
```
import logging

logger = logging.getLogger(__name__)
```

File: scripts/changelog.py (hunk 1 above)
Comment (ita-sammann) [reply scripts/changelog.py]: ✓

File: scripts/changelog.py (hunk 2 above)
Comment (ita-sammann) [reply scripts/changelog.py]: ✓

File: scripts/changelog.py (hunk 3 above)
Comment (ita-sammann) [reply scripts/changelog.py]: ✓

File: scripts/changelog.py (hunk 4 above)
Comment (ita-sammann) [reply scripts/changelog.py]: Nope :-P

**Pull request merged**